### DISCUSSION TITLE
Compatibility

### DISCUSSION THREAD

**Discussion by morad0990** [Sinytra/Connector#599] [Ideas]
Compatibility
If the mod were compatible with optifine it would be an incredible breakthrough.
Do you intend to make the mod compatible with optifine?

**Comment by Su5eD** [comment]
There is no OptiFine compatibility planned for Connector or FFAPI, as these issues don't come from our side to begin with. I recommend using a suitable alternative, such as Embeddium.

See also: https://github.com/Sinytra/ForgifiedFabricAPI/discussions/56

**Comment by kakashi1464** [comment]
First of all, Optifine is a closed source  project, and cause of that, you don't  know what even you need to fix
Secondly, afaik it uses ancient decompiler which adds more problems

**Reply by kakashi1464** [reply]
Sorry if some of my words were misinformation.
